app/views/carts/index.html.erb

```diff
@@ -13,11 +13,14 @@
     <p><%= link_to "Increase quantity", cart_path(id: item.id, condition: "increase"), action: :update, method: :patch %></p>
     <p><%= link_to "Remove", cart_path(id: item.id), method: :delete %></p>
   </div>
-  <div class="total">
-    <% sum = 0 %>
-    <p>Total: $<%= sum += (item.price * quantity) %></p>
-  </div>
 <% end %>
+<div class="total">
+  <% sum = 0 %>
```

Comment:
@mimilettd I wonder if we can extract this logic into the model/controller. I'm not to concerned about it now, but it could be a possible refactor in the future. The piece that caught my eye is that you instantiate this sum variable inside the view.